package.json

```diff
@@ -20,17 +20,14 @@
   ],
   "license": "SEE LICENSE IN LICENSE",
   "bugs": {
-    "url": "https://github.com/austinhuang0131/discordtel/issues"
+    "url": "https://github.com/dtel-hq/dtel/issues"
   },
-  "homepage": "https://discordtel.austinhuang.me",
+  "homepage": "https://dtel.austinhuang.me",
   "engines": {
     "node": "12"
   },
   "dependencies": {
     "@discoin/scambio": "^2.0.4",
-    "@types/rethinkdb": "^2.3.14",
```

Comment:
Are you sure these are devdeps? While these specific packages are types, RethinkDB/WS aren't listed as dependencies despite being required for running the bot (not just dev)

Reply:
They're not dev deps